library/cluster_vm.py

```diff
@@ -165,6 +165,44 @@
         C(clone)
       - Metadata key must be composed of letters and numbers only
     type: dict
+  pacemaker_meta:
+    description:
+      - Pacemaker "meta" parameter to add
+      - This parameter is optional if the I(command) is C(create) or C(clone)
```

Comment:
State that it is **only used with create or clone command** and that it is optional.
Otherwise this message gives the impression it can be used for other commands.
Same remark for below help strings